### ISSUE TITLE
Upgrade 2.2.2 -> 2.3.0 broken

### ISSUE THREAD

**Issue by bzed** [2ndQuadrant/pglogical#243]
Hi,

the upgrade sql from 2.2.2 to 2.3.0 is broken unfortunately:

```
ALTER TABLE pglogical.replication_set_table
      ADD COLUMN set_nsptarget name NOT NULL
    , ADD COLUMN set_reltarget name NOT NULL;
ALTER TABLE pglogical.replication_set_seq
      ADD COLUMN set_nsptarget name NOT NULL
    , ADD COLUMN set_seqtarget name NOT NULL;
```

Obviously this will fail if the tables are not empty.

A PR will follow.

Thanks for fixing,
Bernd

**Comment by bzed** [comment]
@petere gruesse aus Salzburg ;)

**Comment by klando** [comment]
your fix looks correct, sorry for having missed that. @PJMODOS @koichi-szk  maybe you can consider a merge.
There is still a surprise, I don't think we should have distinct path to upgrade (from 2.2.2 and 2.2.3). Distinct problem.

**Comment by petere** [comment]
This has been fixed in version 2.3.1.
